app/src/test/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordUiRendererTest.kt

```diff
@@ -63,6 +64,51 @@ class TeleconsultRecordUiRendererTest {
     verify(ui).setPatientTookMedicines(Yes)
     verify(ui).setPatientConsented(Yes)
     verify(ui).renderPatientDetails(patient)
+    verify(ui).hideDoneProgress()
+    verifyNoMoreInteractions(ui)
+  }
+
+  @Test
+  fun `when teleconsult record create state is creating, then show done progress`() {
```

Comment:
We can rename the situation for this test to `when the teleconsult record is being created`.